libraries/AP_OSD/AP_OSD_MSP_DisplayPort.cpp

```diff
@@ -77,6 +76,13 @@ uint8_t AP_OSD_MSP_DisplayPort::format_string_for_osd(char* buff, uint8_t size,
 
 void AP_OSD_MSP_DisplayPort::flush(void)
 {
+    if(!_bInitedUart)
+	{
```

Comment:
```suggestion
    if (!_bInitedUart) {
```

... and adjust indenting